week5/netmikolab.py

```diff
@@ -0,0 +1,108 @@
+from netmiko import ConnectHandler
+
+def get_data_from_device(device_params, command):
+    with ConnectHandler(**device_params) as ssh:
+        result_shipintbr = ssh.send_command(command)
+        return result_shipintbr
+
+def get_ip(device_params, intf):
+    data = get_data_from_device(device_params, "sh ip int br")
+    result = data.strip().split("\n")
+    for line in result[1:]:
+        words = line.split()
+        if words[0][0] == intf[0] and words[0][-3:] == intf[1:]:
+            return words[1]
```

Comment:
1. you can use RegEx to filter the line of your select interface.
2. Next, use split() to make each column.
3. Return only the column is store ip adress.